releasetasks/templates/notification/notifications.yml.tmpl

```diff
@@ -0,0 +1,194 @@
+{#
+Notifications via Pulse-Notify service
+The macro call should look like
+    task_notifications(taskname, completed=["releasetasks"], failed=["releasetasks"], exception=["releasetasks"])
+As defined at the top of release_graph.yml.tmpl
+
+If any kwarg is a list, it is assumed to be IDs for notification. If the kwarg is a dict, it is a custom notification configuration
+specific to the task.
+#}
+{% if (completed is not none) or (failed is not none) or (artifact is not none) or (exception is not none) %}
+notifications:
+    {% if completed is iterable %}
+    task-completed:
+        subject: "{{ taskname }} completed"
+        message: "{{ taskname }} has completed successfully! Yay!"
+        ids:
+            {% for id in completed %}
+            - {{ id }}
+            {% endfor %}
+    {% elif completed is mapping %}
```

Comment:
I don't see this used anywhere. Can you delete it in this case to simplify the logic?